# Upstream: https://github.com/microsoft/vcpkg/blob/master/ports/libexif/fix-ssize.patch
# ssize_t is not defined on Windows; replace with ptrdiff_t from <stddef.h>

diff --git a/libexif/exif-loader.c b/libexif/exif-loader.c
index ad4ae96..8a24ef6 100644
--- a/libexif/exif-loader.c
+++ b/libexif/exif-loader.c
@@ -19,6 +19,11 @@
  */
 
 #include <config.h>
+#ifdef _WIN32
+#include <stddef.h>
+#define ssize_t ptrdiff_t
+#endif
+
 
 #include <libexif/exif-loader.h>
 #include <libexif/exif-utils.h>
